maquetado/scss/titles.scss

```diff
@@ -0,0 +1,19 @@
+.main-title {
+  align-items: center;
+  display:  flex;
+  font-size: $text-big;
+  
+  &:before {
+    background: url('../assets/triangle.svg')  no-repeat;
+    content: '';
+    display:inline-block;
```

Comment:
Missing a space here. And why it is necessary this?

Reply:
it's really not necessary.